src/components/TagList/_tag-list.scss

```diff
@@ -0,0 +1,46 @@
+@import 'carbon-components/scss/globals/scss/vars';
+@import 'carbon-components/scss/globals/scss/colors';
+@import 'carbon-components/scss/globals/scss/mixins';
+@import 'carbon-components/scss/globals/scss/typography';
+@import 'carbon-components/scss/globals/scss/spacing';
+@import 'carbon-components/scss/globals/scss/import-once';
+@import 'carbon-components/scss/globals/scss/css--typography';
+@import '../Tag/tag';
+
+@include exports('cloud-tag-list') {
```

Comment:
cc @carbon-design-system/design should we be looking for our spacing constants in this file, or is it okay if they don't align?